spring-cloud-gcp-data-datastore/src/main/java/org/springframework/cloud/gcp/data/datastore/core/DatastoreOperations.java

```diff
@@ -67,7 +84,7 @@
 	 * Delete all entities of a given domain type.
 	 * @param entityClass the domain type to delete from Cloud Datastore.
 	 */
-	void deleteAll(Class<?> entityClass);
+	long deleteAll(Class<?> entityClass);
```

Comment:
Please add to javadoc the fact that it returns the count.

Reply:
done